src/plugins/mer/mersettings.cpp

```diff
@@ -21,22 +21,22 @@
 ****************************************************************************/
 
 #include "mersettings.h"
+#include "merconstants.h"
```

Comment:
The original order of includes is reverse compared to the recommendation from https://wiki.qt.io/Coding_Conventions#Including_headers, but now after moving this up it is complete mess :) Any reason to do so?

Reply:
Yes, I agree. What I meant in this case is that we should always follow the (possibly wrong) conventions used by the surrounding code - better a wrong convention than no convention. If a convention needs to be replaced, it should be applied globally and with dedicated commit.